app-header/app-header.html

```diff
@@ -217,7 +217,13 @@
           type: Boolean,
           reflectToAttribute: true,
           value: false
+        },
+
+        blockScrollEffects: {
```

Comment:
As discussed, let's rename `AppScrollEffectsBehavior.noTransitions` to `.disabled` and use that to stop calls to `_updateScrollState`